tests/e2e/Services/Functions/FunctionsCustomServerTest.php

```diff
@@ -502,6 +503,120 @@ public function testCreateFunctionAndDeploymentFromTemplate()
         $function = $this->cleanupFunction($functionId);
     }
 
+    public function testCreateFunctionAndDeploymentFromTemplateBranch()
+    {
+        $starterTemplate = $this->getTemplate('starter');
+        $this->assertEquals(200, $starterTemplate['headers']['status-code']);
+
+        $phpRuntime = array_values(array_filter($starterTemplate['body']['runtimes'], function ($runtime) {
```

Comment:
just $runtime would be enough here. or if needed to be specific, let switch to $nodeRuntime